cloud: support generating endpoints automatically (#14244)
pingcap/docs: tidb-cloud/data-service-get-started.md

> On the right pane, click the **Properties** tab and set properties for the endpoint, such as:

Author: The following changes simplify the get started doc as these limitations will be prompted if users input an invalid value.

- **Path**: the path that users use to access the endpoint. The combination of the request method and the path must be unique within a Data App.

- **Endpoint URL**: (read-only) the URL is automatically generated based on the region where the corresponding cluster is located, the service URL of the Data App, and the path of the endpoint. For example, if the path of the endpoint is `/my_endpoint/get_id`, the endpoint URL is `https://<region>.data.tidbcloud.com/api/v1beta/app/<App ID>/endpoint/my_endpoint/get_id`.

- **Request Method**: the HTTP method of the endpoint. You can use `GET` to retrieve data, use `POST` to create or insert data, use `PUT` to update or modify data, and use `DELETE` to delete data.

- **Timeout(ms)**: the timeout for the endpoint.

- **Max Rows**: the maximum number of rows that the endpoint can operate or return.

> - **Batch Operation**: Controls whether to enable an endpoint with the `POST`, `PUT`, or `DELETE` request method to operate in a batch mode. The batch mode allows you to operate on multiple rows in a single request.

Author: ```suggestion
- **Batch Operation**: Controls whether to enable the endpoint to operate in batch mode. When it is enabled, you can operate on multiple rows in a single request. To enable this option, make sure that the request method of the endpoint is `POST`, `PUT`, or `DELETE`, and that the last SQL statement for the endpoint is an `INSERT`, `UPDATE`, or `DELETE` operation.
```

> - **Batch Operation**: Controls whether to enable the endpoint to operate in batch mode. When it is enabled, you can operate on multiple rows in a single request. To enable this option, make sure that the request method is `POST`, `PUT`, or `DELETE`, and that the last SQL statement for the endpoint is an `INSERT`, `UPDATE`, or `DELETE` operation.

Reviewer: 同上

> - **Batch Operation**: Controls whether to enable the endpoint to operate in batch mode. When it is enabled, you can operate on multiple rows in a single request. To enable this option, make sure that the request method is `POST`, `PUT`, or `DELETE`, and that the last SQL statement for the endpoint is an `INSERT`, `UPDATE`, or `DELETE` operation.

Author: ```suggestion
- **Batch Operation**: Controls whether to enable the endpoint to operate in batch mode. When it is enabled, you can operate on multiple rows in a single request. To enable this option, make sure that the request method is `POST`, `PUT`, or `DELETE`.
```

    - `GET`: use this method to query data, such as a `SELECT` statement.

>     - `POST`: use this method to insert data, such as an `INSERT` statement.

Author: ```suggestion
    - `POST`: use this method to insert or create data, such as an `INSERT` statement.
```

>     - `PUT`: use this method to update data, such as an `UPDATE` statement.

Author: ```suggestion
    - `PUT`: use this method to update or modify data, such as an `UPDATE` statement.
```
